test/test_cuda.py

```diff
@@ -5125,6 +5135,132 @@ def cb(device, alloc, device_alloc, device_free):
         self.assertTrue(x)
 
 
+MIN_BLOCK_SIZE = 512
```

Comment:
nit: lets only have the constants that we use here. If these tests rely on these constants not changing it will be easier to spot.

Reply:
applying in final pr of stack